head/core/tasks.cpp

```diff
@@ -48,16 +54,22 @@ void head_tasks::start_tasks(
     motor_driver::MotorDriver& left_motor_driver,
     motion_controller::MotionController<lms::LeadScrewConfig>&
         right_motion_controller,
-    motor_driver::MotorDriver& right_motor_driver) {
+    motor_driver::MotorDriver& right_motor_driver,
+    presence_sensing_driver::PresenceSensingDriver& presence_sensing_driver) {
     // Start the head tasks
     auto& can_writer = can_task::start_writer(can_bus);
     can_task::start_reader(can_bus);
 
+    auto& presence_sensing = presence_sensing_driver_task_builder.start(
+        5, presence_sensing_driver, head_queues);
```

Comment:
is the `5` here the task priority? feels a bit too prioritized if so

Reply:
5 feels reasonable, given
`#define configMAX_PRIORITIES (7)
`
head/firmware/FreeRTOSConfig.h